docs/how-to/tensorboard.txt

```diff
@@ -51,6 +51,20 @@ TensorBoard for multiple experiments use
   metrics from persistent storage. It may take up to 5 minutes for TensorBoard
   to receive data and render visualizations.
 
+Specifying a Command Config
+---------------------------
+
+Determined supports initializing TensorBoard with a config file. This
+feature can be useful for running TensorBoard with a specific container
+image or with a different resource allocation. Configuration settings
+can be found in the  :ref:`command-notebook-configuration`.
+
+To launch Tensorboard with a config file, use
+``det tensorboard start <experiment-id> --config-file=my_config.yaml``.
```

Comment:
Add an example of a configuration customization we expect to be somewhat common?